src/HermsManagement.py

```diff
@@ -0,0 +1,76 @@
+from discord.ext import commands
+import logging
+import discord
+  
+class HermsManagement(commands.Cog):
+  def __init__(self,bot):
+    self.bot = bot
+    
+  
+  @commands.command(name='a',brief='adds a user to the hermables list',aliases=['add'])

```

Comment:
Same here, always put spaces after commas. Looks like all your decorators need to be fixed.

Reply:
Also, for user-facing stuff, use "friend" instead of "hermable" (I'll probably end up changing the code to use "friend" instead of "hermable" too